File: benchmark/php/arrsum.php
<?php
$arr = [];
for ($i = 0; $i < 10000; $i++) { $arr[] = $i; }
$start = hrtime(true);
$sum = 0;
for ($i = 0; $i < 10000; $i++) { $sum += $arr[$i]; }
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "Sum: " . $sum . "
";
echo "Time: " . round($ms) . "ms
";

File: benchmark/python/arrsum.py
import time

arr = []
for i in range(10000):
    arr.append(i)
sum = 0
start = time.time() * 1000
for i in range(10000):
    sum = sum + arr[i]
end = time.time() * 1000
print(f"Sum: {sum}")
print(f"Time: {end - start:.0f}ms")
